src/nile/deployments.py

```diff
@@ -28,10 +28,18 @@ def register(address, abi, network, alias):
         fp.write("\n")
 
 
-def unregister(address_or_class_hash, network, alias, abi=None):
+def unregister(address_or_class_hash, network, alias, abi=None, is_deployment=True):
```

Comment:
```suggestion
def unregister(address_or_class_hash, network, alias, abi=None, is_declaration=False):
```
since the flag will be passed for declarations, i think we should name it like this for clarity